src/components/Element/Navbar.jsx

```diff
@@ -1,14 +1,10 @@
 import clsx from 'clsx';
-import { Logo, MenuIcon, Menu } from './NavbarElement';
+import { Logo, Menu } from './NavbarElement';
 
 export function Navbar() {
   return (
-    <nav className={clsx('relative top-0 left-0', 'px-3 py-6', 'text-center')}>
+    <nav className={clsx(' h-full', 'bg-[#323232]', 'px-3 py-6', 'text-center')}>
```

Comment:
we use tailwind, can you import this to tailwind config so its reusable

Reply:
I just tried the new layout with this bg color temporarily, I remove to config right now